php/libraries/LorisForm.class.inc

```diff
@@ -1180,7 +1180,7 @@ class LorisForm
 
                 if (isset($el['required'])
                     && $el['required'] === true
-                    && empty($value)
+                    && (is_null($value) || (empty($value) && $value !=0))
```

Comment:
@driusan suggests that we should just use `$value === ""` instead of this, @sruthymathew123 can you see if that works?